docs/i18n/portuguese/how-to-work-on-coding-challenges.md

```diff
@@ -349,80 +352,80 @@ class MyComponent extends React.Component {
 
-### Adding new Challenge hints/solutions Topics
+### Adicionando um novo tópico de dicas/soluções em um desafio
 
-Take the following steps when adding a new challenge hints/solutions related topic.
+Faça o passo-a-passo a seguir quando for adicionar novos tópicos de dicas/soluções relacionadas a um desafio.
 
 1. Start by following the same steps for creating a new topic but review the next for creating the title.
-2. The title of the topic should start with `freeCodeCamp Challenge Guide:` concatenated with the actual title of the curriculum challenge. For example, if the challenge is named "`Chunky Monkey`", the topic title would be "`freeCodeCamp Challenge Guide: Chunky Monkey`".
+2. O título do tópico deve começar com `freeCodeCamp Challenge Guide:` concatenado com o título atual do desafio de currículo. Por exemplo, se o desafio é chamado "`Chunky Monkey`", o título do tópico seria "`freeCodeCamp Challenge Guide: Chunky Monkey`".
 3. `camperbot` should be the owner of these topics/posts, so you will need to request an admin to change the ownership of the main post to `camperbot`.
-4. Once the new topic is created, a forum topic id is created. It is located at the end of the forum topic URL. This id must be added to the frontmatter of the curriculum challenge file via the normal pull request process for the `Get a Hint` button to link to the topic.
+4. Depois que o novo tópico for criado, será criado um id para o tópico do fórum. It is located at the end of the forum topic URL. This id must be added to the frontmatter of the curriculum challenge file via the normal pull request process for the `Get a Hint` button to link to the topic.
 
-### Guidelines for content of hints and solutions topics
+### Diretrizes para o conteúdo dos tópicos de dicas e soluções
 
-When proposing a solution for a curriculum challenge related Guide topic, the full code must be added. This includes all the original seed code plus any changes needed to pass all the challenge tests. The following template should be used when creating new hints/solutions topics:
+Ao propor uma solução para um tópico do guia relacionado a um desafio de currículo, o código completo deve ser adicionado. Isso inclui todo o código seed original, assim como as alterações necessárias para passar em todos os testes do desafio. O modelo a seguir deve ser usado ao criar um novo tópico de dicas/soluções:
 
 ````md
-# Challenge Name Goes Here
+# O nome do desafio fica aqui
 
 ---
 
-## Problem Explanation
+## Explicação do problema
 
-This summarizes what needs to be done without just restating the challenge description and/or instructions. This is an optional section
+Resume o que precisa ser feito sem copiar a descrição do desafio e/ou instruções. Essa é uma seção opcional
 
-#### Relevant Links
+#### Links relevantes
 
-- [Link Text](link_url_goes_here)
-- [Link Text](link_url_goes_here)
+- [Texto do link](url_do_link_fica_aqui)
```

Comment:
I am not sure why this double line has the same message.

Reply:
This is correct, though - the template is for adding new Guide posts, and is showing how to structure the list of relevant links.

Reply:
@nhcarrigan, oh ok, I just saw two same example markdown links if you are fine with that.